src/map/map.ts

```diff
@@ -102,6 +105,50 @@ const DEFAULT_CONFIG = {
         'hoverClosest3d',
         'tableRotation',
         'resetCameraLastSave3d',
+        'toImage',
+    ],
+
+    modeBarButtonsToAdd: [
+        {
+            name: 'Download as PNG',
+            icon: {
+                width: 400,
+                height: 424,
+                path: extractSvgPath(PNG_SVG),
+            },
+            click: function (gd: PlotlyScatterElement) {
+                Plotly.downloadImage(gd, {
+                    filename: 'newplot',
+                    format: 'png',
+                    width: gd._fullLayout.width,
+                    height: gd._fullLayout.height,
```

Comment:
This is a bit small, and the plot ends up pixelated. I think we should multiply the width/height by 2, and make sure this is at least 600px in each direction.

Reply:
This proved to be harder than expected =( Changing the size of the plot also changes the spacing between points; and using the `scale` parameter does scale the plot up; but the resulting plot is still pixelated. I think this is because we are using `scattergl` trace, and plotly does not re-render the plot on a large canvas, but instead use the already existing one.